packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/object-method-with-arrows/expected.js

```diff
@@ -1,4 +1,5 @@
 class Class {
+  /*#__PURE__*/
```

Comment:
I'm not sure about this one :thinking:

Reply:
Should I add a check for function declarations before adding the annotation?